dashboard/package.json

```diff
@@ -100,7 +103,7 @@
     "angular-animate": "1.4.8",
     "angular-aria": "1.4.8",
     "angular-cookies": "1.4.8",
-    "angular-file-upload": "2.0.0",
+    "angular-file-upload": "^2.5.0",
```

Comment:
FYI dependencies need Eclipse CQ to be updated

Reply:
Thank you for your information. I didn't know about CQ and I'm browsing [IPZilla]() now.
Hmm... I see it's better to upgrade in step-by-step (First, just upgrade CQ passed only. Next, apply CQs) approach.